lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -781,6 +784,96 @@ def get(self, request, *args, **kwargs):
         return render_to_response(self.template_name, context)
 
 
+class MgmtCommands(SysadminDashboardView):
+    """This class provides a view to handle the execution of management commands"""
```

Comment:
These class-level docstrings are more verbose than needed. This is a
common trend (I know I used to do this too), to describe a `class`
docstring by starting with `This class does...`. We can strike that
entire bit; we already know, explicitly, that this is a `class`.

```python
class MgmtCommands(SysadminDashboardView):
    """
    Render views for management commands
    """

class ExecuteCommand(SysadminDashboardView):
    """
    Handle AJAX requests to execute managment commands
    """
```